src/Splat.Log4Net/Log4NetLogger.cs

```diff
@@ -24,11 +25,19 @@ public sealed class Log4NetLogger : ILogger
         public Log4NetLogger(global::log4net.ILog inner)
         {
             _inner = inner ?? throw new ArgumentNullException(nameof(inner));
+            SetLogLevel();
+            _inner.Logger.Repository.ConfigurationChanged += OnInnerLoggerReconfigured;
```

Comment:
Is there any possibility this would throw a null reference exception?

Reply:
in theory it's possible because repository is

```
		virtual public Hierarchy Hierarchy
		{
			get { return m_hierarchy; }
			set { m_hierarchy = value; }
		}

		public ILoggerRepository Repository
		{ 
			get { return m_hierarchy; }
		}
```

So while we're in theory talking to the parent of the logger, it could be missing. but we can play nicely.